cmd/conduit/main.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/algorand/indexer/conduit"
+	"github.com/algorand/indexer/config"
+)
+
+import (
+	_ "github.com/algorand/indexer/exporters/all"
+	_ "github.com/algorand/indexer/importers/all"
+	_ "github.com/algorand/indexer/processors/all"
+)
```

Comment:
Nit: I think these can all be in the main import block. If you're going to separate them at least put a comment saying why they're important and not to remove them.